tox.ini

```diff
@@ -69,7 +69,7 @@ deps =
 
     # Figure tests need a tightly controlled environment
     figure-!devdeps: astropy==4.0.1.post1
-    figure-!devdeps: matplotlib==3.2.1
+    figure-!devdeps: matplotlib==3.2.2
```

Comment:
This needs bumping to remove a pandas deprecation warning; it might still need bumping up to 3.3 to remove that, but will see what happens.

Reply:
Yep, bumping the minor-minor number worked fine.